test/RibbonThetaVault.ts

```diff
@@ -229,6 +241,70 @@ describe("RibbonThetaVault", () => {
     },
     availableChains: [CHAINID.ETH_MAINNET],
   });
+
+  behavesLikeRibbonOptionsVault({
+    name: `Ribbon NEAR Theta Vault (Call)`,
+    tokenName: "Ribbon NEAR Theta Vault",
+    tokenSymbol: "NEAR-THETA",
+    asset: NEAR_ADDRESS[chainId],
+    assetContractName: "IWBTC",
+    strikeAsset: USDC_ADDRESS[chainId],
+    collateralAsset: NEAR_ADDRESS[chainId],
+    chainlinkPricer: CHAINLINK_NEAR_PRICER[chainId],
+    deltaFirstOption: BigNumber.from("1000"),
+    deltaSecondOption: BigNumber.from("1000"),
+    deltaStep: BigNumber.from(chainId === CHAINID.ETH_MAINNET ? "100" : "10"),
+    depositAmount: parseEther("1"),
+    minimumSupply: BigNumber.from("10").pow("10").toString(),
+    expectedMintAmount: BigNumber.from("100000000"),
+    premiumDiscount: BigNumber.from("997"),
+    managementFee: BigNumber.from("2000000"),
+    performanceFee: BigNumber.from("20000000"),
+    auctionDuration: 21600,
+    tokenDecimals: 24,
+    isPut: false,
+    isUsdcAuction: false,
+    gasLimits: {
+      depositWorstCase: 101000,
+      depositBestCase: 90000,
+    },
+    mintConfig: {
+      contractOwnerAddress: NEAR_OWNER_ADDRESS[chainId],
+    },
+    availableChains: [],
+  });
+
+  behavesLikeRibbonOptionsVault({
+    name: `Ribbon AURORA Theta Vault (Call)`,
+    tokenName: "Ribbon AURORA Theta Vault",
+    tokenSymbol: "AURORA-THETA",
+    asset: AURORA_ADDRESS[chainId],
+    assetContractName: "IWBTC",
+    strikeAsset: USDC_ADDRESS[chainId],
+    collateralAsset: AURORA_ADDRESS[chainId],
+    chainlinkPricer: CHAINLINK_AURORA_PRICER[chainId],
+    deltaFirstOption: BigNumber.from("1000"),
+    deltaSecondOption: BigNumber.from("1000"),
+    deltaStep: BigNumber.from(chainId === CHAINID.ETH_MAINNET ? "100" : "10"),
```

Comment:
should we do delta step of $1 for Aurora (at $27 right now)

Reply:
I changed this to $5 instead of $1